### DISCUSSION TITLE
ScriptReference/Awaitable.NextFrameAsync

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#510] [Notes]
# ScriptReference/Awaitable.NextFrameAsync

https://docs.unity3d.com/ScriptReference/Awaitable.NextFrameAsync.html

**Comment by vertxxyz** [comment]
When used in the editor, this function will wait until [`PlayerLoop.Update`](https://docs.unity3d.com/ScriptReference/PlayerLoop.Update.html)/[`[ExecuteAlways]`](https://docs.unity3d.com/ScriptReference/ExecuteAlways.html) (not the same as [`EditorApplication.update`](https://docs.unity3d.com/ScriptReference/EditorApplication-update.html)).
